### ISSUE TITLE
Remember me checkbox is not working (always ON)

### ISSUE THREAD

**Issue by omgitstom** [stormpath/idsite-src#4]
Regardless if remember me is checked or not the cookie that is configured with the idle timeout and max age is still being applied.

**Comment by lprimak** [comment]
Agreed.  The checkbox has no effect, but it's state should be communicated to the application as one of the parameters via stormpathCallback

**Comment by lprimak** [comment]
Any updates on this issue?  We would like to see this implemented.
Thank you

**Comment by lprimak** [comment]
Still waiting on this...thanks!

**Comment by mdeggies** [comment]
We've received interest via support in this feature. Would be nice to see on the board :)

**Comment by deltacar** [comment]
Any progress on this feature? 
Is there any chance we can have it this year?
Thank you in advance.

**Comment by mdeggies** [comment]
There is renewed (enterprise) interest in this. Can we provide an ETA?

**Comment by heatherita** [comment]
Is this functionality likely to be available in Okta, or should I just remove it?

**Comment by mdeggies** [comment]
@heatherita, Okta does not offer hosted registration/login pages like Stormpath's ID Site. The closest thing to it is the sign in widget: https://developer.okta.com/code/javascript/okta_sign-in_widget. You can host this widget on a separate domain if you'd like, or embed it directly on your site. It does have a remember me button that works, which you can use.
